packages/react-chat/.babelrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100,
+          "safari": 15,
+          "firefox": 91
+        }
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```

Comment:
this .babelrc was generated via `npx storybook@latest babelrc`

before adding this, got the same issue as featured below (link to the solution)
https://github.com/storybookjs/storybook/issues/22123#issuecomment-1514124086

Reply:
I had node v16